src/components/transaction-list/TransactionList.js

```diff
@@ -325,7 +320,6 @@ export default function TransactionList({
         avatarOptions={avatarOptions}
         darkMode={isDarkMode}
         data={data}
-        isAvatarPickerAvailable={isAvatarPickerAvailable}
```

Comment:
is this one more place we want `isAvatarPickerAvailable` truthy

Reply:
here, this is not needed, because this defaults to true on the native list on iOS!